main/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">EverTranslator (ترجمة الشاشة)</string>
+    <string name="translation_provider_microsoft_azure">"مايكروسوفت أزور "</string>
+    <string name="translation_provider_google_ml_kit">Google MLKit</string>
+    <string name="translation_provider_google_translate_app">تطبيق ترجمة جوجل</string>
+    <string name="translation_provider_none">استخراج النص فقط</string>
+    <string name="translation_provider_none">"استخراج النص "</string>

```

Comment:
The key `translation_provider_none` is listed double times on line 7 and 8.
Please remove one of them, thank you!

Reply:
thank you I don't know how this happened